action/QuadMove.action

```diff
@@ -0,0 +1,11 @@
+#goal definition
+string movement_type
+bool preempt
+float64 takeoff_height
+---
+#result definition
+bool success
```

Comment:
Why should the caller know if it was preempted? Is that adding more functionality than we need now and for a while?